### PR TITLE
Docs: remove IRC remnants

### PR DESCRIPTION
**Pull request by Andersson007**
Docs: remove IRC remnants https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812/26?u=andersson007

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -9,10 +9,9 @@ body:
       value: |
         **Thank you for reporting a bug in Ansible Builder.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-builder](https://web.libera.chat/#ansible-builder)
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
        for information on how to get in touch.
```

File: .github/ISSUE_TEMPLATE/documentation_report.yml [hunk 2]
```diff
@@ -10,11 +10,9 @@ body:
...
         **Thank you reporting an issue with Ansible Builder documentation.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-builder](https://web.libera.chat/#ansible-builder)
-
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/documentation_report.yml]:
```suggestion
        for information on how to get in touch.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -10,11 +10,9 @@ body:
...
         **Thank you for suggesting a new feature for Ansible Builder.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-builder](https://web.libera.chat/#ansible-builder)
-
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/feature_request.yml]:
```suggestion
       for information on how to get in touch.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,7 +1,8 @@
 # Ansible Builder Contributing Guidelines
 
-If you have questions about this document or anything not covered here, come
-chat with us at `#ansible-builder` on irc.libera.chat.
+If you are looking for community support, please visit
+the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+for information how to reach out to us.
```
Comment (oraNod) [inline CONTRIBUTING.md]:
```suggestion
for information on how to get in touch.
```

**Review by oraNod — COMMENTED** [review]
Thanks @Andersson007 

I've left some suggestions. Prepositions are tricky. You could also do `s/get in touch/contact us` to avoid a slightly idiomatic phrase.

**Comment by Andersson007** [comment]
@oraNod committed, thanks, PTAL

**Review by oraNod — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -9,10 +9,9 @@ body:
       value: |
         **Thank you for reporting a bug in Ansible Builder.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-builder](https://web.libera.chat/#ansible-builder)
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/builder/en/latest/community/)
+        for information on how to get in touch.
```
Comment (Shrews) [inline .github/ISSUE_TEMPLATE/bug_report.yml:14]: The line beginning with "for" seems to be under indented when I download the change locally and inspect it (causing the YAML linting error you see in CI).

**Review by Shrews — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 5 above)
Comment (Andersson007) [reply .github/ISSUE_TEMPLATE/bug_report.yml:14]: @Shrews Fixed, thanks! PTAL

**Review by Shrews — APPROVED** [review]

**Pull request merged**